### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by miniuchi**

### CODE REVIEW

File: build/Build.pm [hunk 1]
```diff
@@ -510,7 +510,6 @@ sub stage_distro {
     }
 
     # Un-tar the distribution.
-    my $tar;
```
Comment (charsbar) [inline build/Build.pm]: might be better to comment out?
(see L.520-521)

File: t/model/161-category.t [hunk 2]
```diff
@@ -64,7 +64,7 @@ MT::Association->link( $ukawa,  $designer,        $blog );
 MT::Association->link( $ichikawa, $edit_categories, $second_blog );
 
 # Run tests
-my ( $app, $out );
```
Comment (charsbar) [inline t/model/161-category.t]: This $app is also unused because of my $app in L75.

File: build/Build.pm (hunk 1 above)
Comment (miniuchi) [reply build/Build.pm]: https://github.com/movabletype/movabletype/pull/776/commits/bc0e43e597f1246e8221472b099ea08302688013

File: t/model/161-category.t (hunk 2 above)
Comment (miniuchi) [reply t/model/161-category.t]: https://github.com/movabletype/movabletype/pull/776/commits/a6aad59f55a450e2027041959a5f92f47b6c9dff

**Pull request merged**